src/build/dotnet/jobs/nuget-package.job.yaml

```diff
@@ -1,26 +1,110 @@
+# Job to build a NuGet Package that will be hosted on an internal VSTS feed
 parameters:
-  projects: '**/*.csproj'
-  tests: '**/*Tests/*.csproj'
-  displayName: 'Build Package'
-  skipDependencyChecks: false
-  signAssemblies: true
-  releaseBranchName: 'master'
-  packageSource: 'internalPublic'
+  - name: displayName
+    default: 'Build Package'
+  - name: projects
+    default: '**/*.csproj'
+  - name: tests
+    default: '**/*Tests/*.csproj'
+  - name: skipDependencyChecks
+    default: false
+  - name: signAssemblies
```

Comment:
if this is true, why can't we use `signed-nuget-package.job.yaml`?

Reply:
Plan to leave this as-is for now, #181379 has been raised on the Internal Development backlog to investigate and implement.